### ISSUE TITLE
export does not work on blender 4.2.3

### ISSUE THREAD

**Issue by warriorman8** [tkeskita/snappyhexmesh_gui#19]
please fix.

**Comment by tkeskita** [comment]
I installed and tested with Blender 4.2.3, but could not reproduce your problem. Export seems to work OK for me.

If you meant Blender version 4.3, that is not supported, as it is not an LTS release.

**Comment by Lourencosm** [comment]
That happened to me when I used projections.
The issue was resolved by altering the following in the utils.py file.
Where:
`bpy.ops.export_mesh.stl(filepath = path + '/{}.stl'.format(o), use_selection=True)`
Replace with:
`bpy.ops.wm.stl_export(filepath = path + '/{}.stl'.format(o), export_selected_objects=True)`

**Comment by Lourencosm** [comment]
For me, it works perfectly with blender 4.2.3 in windows 11, OpenFoam.com.

**Comment by tkeskita** [comment]
Yes I exported openfoam.com fork and I use Linux. Can you give any more details about how it fails? If you start Blender from the terminal, is there some output about error in the terminal when you press Export button?

**Comment by Lourencosm** [comment]
> Anyway is it possible you create a cf-mesh version?

That would be very interesting! Would CF-Mesh support that?

**Comment by Lourencosm** [comment]
I mean financial support from CF-Mesh to @tkeskita develop the add-on.

**Comment by tkeskita** [comment]
Currently I have far too much on my TODO list to explore cfMesh
